lib/util/exec/docker/index.ts

```diff
@@ -1,49 +1,85 @@
-import { uniq } from 'lodash';
-
 type Opt<T> = T | null | undefined;
 
+type VolumesPair = [string, string];
+export type VolumeOption = Opt<string> | Opt<VolumesPair>;
```

Comment:
Do we have any instances where we want to map A:B instead of A:A? I don't think we want to support this actually, for some architectural reasons related to nested Docker. If we don't have any active use cases then I want to remove any possibility of adding it by limiting this to be strings only that are always mapped A:A.

Reply:
Yes, here: https://github.com/renovatebot/renovate/blob/master/lib/manager/npm/post-update/npm.ts#L87

I don't see how to avoid this case

Reply:
I need to think about this one more. I guess we keep it in for now